deliveries/lastic-milestone-3-4.md

```diff
@@ -0,0 +1,218 @@
+
+* **Application Document:** https://github.com/w3f/Grants-Program/blob/master/applications/lastic-grant3.md
+* **Milestone Number:** 1 and 2
+
+### Context (Optional)
+Lastic's milestones 3 and 4 aims to enhance the functionality and user experience of the Coretime Chain indexing process through Subsquid, along with providing basic visualization capabilities.
+
+### Deliverables
+This section lists all deliverables for Milestone 3 and 4, as outlined in the original application. Each deliverable includes a link to its implementation, with additional notes provided where applicable.
+
+### Milestone 1: Indexing Coretime Chain with Subsquid - Functioning Indexer and Basic Visualization
+
+This milestone focuses on establishing a robust infrastructure for indexing the Coretime Chain using Subsquid, along with the initial steps towards visualizing Coretime data. Each deliverable is detailed below, complete with relevant links to implementations and additional notes for clarity.
+
+| Number   | Deliverable                                      | Link                                                                                                    | Notes                                                                                                     |
+|----------|--------------------------------------------------|---------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------|
+| **0a.**  | License                                          | [subsquid-sdk LICENSE](https://github.com/LasticXYZ/squid-sdk/blob/main/LICENSE) and [subsquid LICENSE](https://github.com/LasticXYZ/subsquid/blob/main/LICENSE) | The project is licensed under GPLv3, ensuring open-source accessibility and compliance. |
+| **0b.**  | Documentation                                    | [subsquid-sdk README.md](https://github.com/LasticXYZ/squid-sdk) and [subsquid README.md](https://github.com/LasticXYZ/subsquid)| Provides extensive documentation in README and dedicated docs site, facilitating user understanding. |
+| **0d.**  | Error Logging                                    | [Lastic UI/issues](https://github.com/LasticXYZ/LasticUI/issues)                                        | Errors are thoroughly documented as issues on GitHub, set for future resolution. We also set up a community bounty program. |
+| **0d.**  | Article Publishing                               | [Article link](https://blog.lastic.xyz/revolutionizing-blockchain-interactivity-unveiling-lastics-groundbreaking-developments-042225bb0b72) | Discusses Lastic's latest breakthroughs, enhancing blockchain interaction.    |
```

Comment:
@poppyseedDev just a reminder to follow our [announcement guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/announcement-guidelines.md) and submit the draft article to the grants pr address as well, so we can proof-read it and consider it for promotion on W3F's X profile. 👍